products/focus/testing_focus.md

```diff
@@ -0,0 +1,38 @@
+# Testing on Focus for iOS and Focus for Android for a new release
+
+Given Focus for iOS and Focus for Android have very similar test procedures, this document will cover both at once.
+
+Localizers currently mostly rely on screenshots for testing. Latest information and updates about testing are provided on the [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n), which you should follow if you are working on l10n for any of the existing Mozilla products.
+
+## Testing with screenshots
+
+Screenshots are provided by the mobile teams and sent out on dev.l10n. The latest iterations can be found [right here for Focus for Android](https://npark-mozilla.github.io/index.html), [right here for Focus for iOS (iPhone5)](https://drive.google.com/drive/folders/0BwM0URyvbEeSQmZPVGdiWVlfNm8) and [right here for Focus for iOS (iPhone7)](https://drive.google.com/drive/folders/0BwM0URyvbEeSQmZPVGdiWVlfNm8).
+
+## Test builds
+
+While there are testing builds available via BuddyBuilds, the process is currently manual and localizers should reach out to the l10n mobile project manager in order to be added to these builds (currently **delphine** at **mozilla** dot **com**).
+For Focus for iOS, please note an iOS device running v9 or above is needed. There is no specified version needed for Android.
+
+## Aspects to review
+
+Here is a list of issues you should try to identify when testing the build:
+* Language quality.
+* Truncated words (cut-off from screen).
+* Anything that appears broken on the UI.
+* Check out at all the main screens, UI, menus, tabs, new features, etc. Make sure these all look good, that everything is properly translated and appears as expected.
+* Font support.
+* Untranslated content.
+
+Concerning untranslated content on Focus for iOS: please note that we currently have two tiers of support in iOS. If your locale is only in the Tier 2 support (which corresponds to the languages under "Other Languages" in the iOS system settings), then it sometimes happens that menu items, which are part of the OS itself, may not be localizable - and will therefore unfortunately appear in English on the final UI.
```

Comment:
it's dialogs more than menu items, or at least mention both
English -> Primary OS language (no clue how it's called in iOS, but if you set OS to French and Fulah as Other language, it will be in French).